src/app-layer-htp-range.c

```diff
@@ -66,23 +68,33 @@ static int ContainerUrlRangeSet(void *dst, void *src)
     HttpRangeContainerFile *dst_s = dst;
     dst_s->len = src_s->len;
     dst_s->key = SCMalloc(dst_s->len);
-    BUG_ON(dst_s->key == NULL);
+    if (dst_s->key == NULL)
```

Comment:
This was some copied/pasted code, so I guess the original can have the same improvements...

Reply:
Agreed. Can you do that or at least create a ticket for it to track it?

Reply:
Looks like there is only one occurence for `git grep "BUG_ON(dst_s"` in datasets-string.c `StringSet`

Should I create add a commit on top of #6308 and create a new PR ?